### PR TITLE
Submenu trending topic

### PR DESCRIPTION
**Pull request by ljubar**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | no
| New feature?              | yes
| Deprecations?             | no
| Changelog update required | no
| License                   | AGPLv3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,25 +0,0 @@
-/app/bootstrap.php.cache
```
Comment (takeit) [inline .gitignore]: why did you remove that? it shouldnt be removed at all

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/trending.html.twig [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <section id="content">
+
+            {% set trendingTopic = app.request.get('topic') %}
+            <h2>Topic '{{ trendingTopic }}'</h2>
+
+            {# Setup list and pagination parameters #}
+            {% set itemsPerPage, currentPage = 8, app.request.get('page', 1) %}
+            {% set start = ((currentPage - 1) * itemsPerPage) %}
+
+            {% gimmelist article from articles|order('publishedAt','desc')|start(start)|limit(itemsPerPage) with {keyword: trendingTopic} %}
```
Comment (takeit) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/trending.html.twig]: shouldnt this be `keywords`?

Comment (ljubar) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/trending.html.twig]: technically they are keywords but we present them here as topics

File: .gitignore [hunk 3]
```diff
@@ -22,4 +22,5 @@
 /web/config.php
 logs/
 build/
+/src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/sftp-config.json
```
Comment (takeit) [inline .gitignore]: this shouldnt be here, if you want to ignore it, you need to add it to your local global gitignore file not to the repository
